BENCHMARKS.md

```diff
@@ -104,3 +104,10 @@ You need to install the [Roboto Font](https://fonts.google.com/specimen/Roboto)
 
 The inclusion of this `BENCHMARKS.md` file was inspired by the excellent benchmarking documentation
 in [Orogene](https://github.com/orogene/orogene/blob/472e481b4fc6e97c2b57e69240bf8fe995dfab83/BENCHMARKS.md).
+
+
+## Troubleshooting
+
+### Flaky benchmarks
+
+If you're seeing high variance in your benchmarks, then it's likely that you're running into throttling or DDoS prevention from your ISP. In that case, ISPs forcefully terminate TCP connections with a TCP reset. We believe this is due to the benchmarks making the exact same requests in a very short time (especially true for `uv`).  A possible workaround is to connect to VPN to bypass your ISPs filtering mechanism.
```

Comment:
Could you add a qualifying word here to indicate that this is about the cold benchmarks and not the warm ones? The warm benchmarks shouldn't suffer from the same problem. Maybe just say, "in your cold benchmarks" or "benchmarks that aren't using a cache."

Reply:
Good point